index.html

```diff
@@ -0,0 +1,367 @@
+
+      <div class="info-video">
+        <h3 class="video-title">
+          how to sleep at the nigth
+        </h3>
+        <span class="views">29,985 views</span>
+      </div>
+    </article>
+    <article class="recommended-video">
+        <img src="https://images.ctfassets.net/hrltx12pl8hq/qGOnNvgfJIe2MytFdIcTQ/429dd7e2cb176f93bf9b21a8f89edc77/Images.jpg" width="100px" height="90px">
+
+        <div class="info-video">
+          <h3 class="video-title">
+            how to sleep at the nigth
+          </h3>
+          <span class="views">29,985 views</span>
+        </div>
+      </article>
+      <article class="recommended-video">
+          <img src="https://images.ctfassets.net/hrltx12pl8hq/qGOnNvgfJIe2MytFdIcTQ/429dd7e2cb176f93bf9b21a8f89edc77/Images.jpg" width="100px" height="90px">
+  
+          <div class="info-video">
+            <h3 class="video-title">
+              how to sleep at the nigth
+            </h3>
+            <span class="views">29,985 views</span>
+          </div>
+        </article>
+        <article class="recommended-video">
+            <img src="https://images.ctfassets.net/hrltx12pl8hq/qGOnNvgfJIe2MytFdIcTQ/429dd7e2cb176f93bf9b21a8f89edc77/Images.jpg" width="100px" height="90px">
+    
+            <div class="info-video">
+              <h3 class="video-title">
+                how to sleep at the nigth
+              </h3>
+              <span class="views">29,985 views</span>
+            </div>
+          </article>
+          <article class="recommended-video">
+              <img src="https://images.ctfassets.net/hrltx12pl8hq/qGOnNvgfJIe2MytFdIcTQ/429dd7e2cb176f93bf9b21a8f89edc77/Images.jpg" width="100px" height="90px">
+      
+              <div class="info-video">
+                <h3 class="video-title">
+                  how to sleep at the nigth
+                </h3>
+                <span class="views">29,985 views</span>
+              </div>
+            </article>
+            <article class="recommended-video">
+                <img src="https://images.ctfassets.net/hrltx12pl8hq/qGOnNvgfJIe2MytFdIcTQ/429dd7e2cb176f93bf9b21a8f89edc77/Images.jpg" width="100px" height="90px">
+        
+                <div class="info-video">
+                  <h3 class="video-title">
+                    how to sleep at the nigth
+                  </h3>
+                  <span class="views">29,985 views</span>
+                </div>
+              </article>
+              <article class="recommended-video">
+                  <img src="https://images.ctfassets.net/hrltx12pl8hq/qGOnNvgfJIe2MytFdIcTQ/429dd7e2cb176f93bf9b21a8f89edc77/Images.jpg" width="100px" height="90px">
+          
+                  <div class="info-video">
+                    <h3 class="video-title">
+                      how to sleep at the nigth
+                    </h3>
+                    <span class="views">29,985 views</span>
+                  </div>
+                </article>
+</div>
```

Comment:
- It could be better if you use HTML5 tags, this could help to perform search engine. 👍 
- **Section** tag could be well suited for your parent content (#268 code line). 😀 
## Check this article: [Section examples](https://html.spec.whatwg.org/multipage/sections.html#the-section-element)